docs/Analyzer Configuration.md

```diff
@@ -392,3 +392,20 @@ Option Values: integral values
 Default Value: Specific to each configurable rule ('100000' by default for most rules)
 
 Example: `dotnet_code_quality.CA5387.sufficient_IterationCount_for_weak_KDF_algorithm = 100000`
+
+#### Do not prefix enum values with type name
```

Comment:
You seem to have added the new option under configuration options for `Dataflow analysis`. You should move the added documentation above this section.